packages/docs-theme/src/components/navigator.tsx

```diff
@@ -34,6 +45,9 @@ export class Navigator extends React.PureComponent<INavigatorProps> {
     public componentDidMount() {
         this.sections = [];
         eachLayoutNode(this.props.items, (node, parents) => {
+            if (Utils.safeInvoke(this.props.itemExclude, node)) {
```

Comment:
to be safe, we should make sure we handle the case this returns 'undefined' instead of 'false'